bundles/org.openhab.core.test/src/main/java/org/openhab/core/test/java/JavaOSGiTest.java

```diff
@@ -160,7 +160,7 @@ protected <T> List<T> getServices(Class<T> clazz, Predicate<ServiceReference<T>>
     private <T> ServiceReference<T> @Nullable [] getServices(final Class<T> clazz) {
         try {
             @SuppressWarnings("unchecked")
-            ServiceReference<T> serviceReferences[] = (ServiceReference<T>[]) bundleContext
+            ServiceReference<T>[] serviceReferences = (ServiceReference<T>[]) bundleContext
```

Comment:
Does this work ?
`ServiceReference<T>[] serviceReferences = (ServiceReference<>[]) bundleContext`

Reply:
just a matter of style, should have same semantics in java